### PR TITLE
add Texas_S_ PVQFN_N-16_EP2.1x2.1mm footprint

### PR DESCRIPTION
**Pull request by nivekg**
Added Texas_RGV_S_ PVQFN_N-16_EP2.1x2.1mm footprint for INA3221.
Symbol-PR: https://github.com/KiCad/kicad-symbols/pull/1343
Footprint-PR: https://github.com/KiCad/kicad-footprints/pull/1641

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 1]
```diff
@@ -1096,3 +1096,56 @@ Texas_VQFN_RGE0024C:
...
+Texas_RGV_S-PVQFN-N16_EP2.1x2.1mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/ina3221.pdf#page=44'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  custom_name_format: 'Texas_RGV_S-PVQFN-N{pincount}_EP{ep_size_x:g}x{ep_size_y:g}mm{vias:s}'
+  body_size_x:
+    minimum: 3.9
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: Should be 3.85/4.15mm. Also below.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 2]
```diff
@@ -1096,3 +1096,56 @@ Texas_VQFN_RGE0024C:
...
+    maximum: 4.1
+  overall_height:
+    minimum: 0.8
+    maximum: 1.0
+
+  lead_width:
+    minimum: 0.23
+    maximum: 0.38
+  lead_len:
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: Should be 0.45-0.65mm.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 3]
```diff
@@ -1096,3 +1096,56 @@ Texas_VQFN_RGE0024C:
...
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [3, 3]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    EP_paste_coverage: 0.5
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: This is rather low. Use 0.65 or I believe you can comment it out for a default value.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml [hunk 4]
```diff
@@ -1096,3 +1096,56 @@ Texas_VQFN_RGE0024C:
...
+    maximum: 0.5
+
+  EP_size_x:
+    nominal: 2.1
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 2.1
+    tolerance: 0.1
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [3, 3]
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: I think `[2, 2]` will work better here with a 3x3 via layout.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml (hunk 3 above)
Comment (nivekg) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: I looks like changing this values results in this error when running the script:

ValueError: One value in ((x=0.8, y=0.8)) too small. Limit is (x=0.8465370635713478, y=0.8465370635713478)

Comment (evanshultz) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: @nivekg
You can allow paste over vias to remove this error. However, you'll then probably want to jack up the paste coverage to 0.75 or 0.8.

@poeschlr 
Are you aware of this issue? Is this something that can be fixed in the script? If you don't know or haven't looked into it do you want me to check it out?

Comment (nivekg) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml]: @evanshultz 
Changes made.

**Comment by evanshultz** [comment]
Footprints are in. This is ready to be merged.

**Pull request merged**